static/app/views/replays/detail/detailLayout.tsx

```diff
@@ -79,32 +93,22 @@ const HeaderPlaceholder = styled(function HeaderPlaceholder(
   background-color: ${p => p.theme.background};
 `;
 
-function EventHeader({event}: {event: Event | undefined}) {
-  if (!event) {
-    return <HeaderPlaceholder width="500px" height="48px" />;
-  }
-
-  const urlTag = event.tags.find(({key}) => key === 'url');
-  const pathname = getUrlPathname(urlTag?.value ?? '') ?? '';
+const HeaderContent = styled(Layout.HeaderContent)`
+  margin-bottom: 0;
+`;
 
-  return (
-    <BigNameUserBadge
-      avatarSize={32}
-      user={{
-        username: event.user?.username ?? '',
-        id: event.user?.id ?? '',
-        ip_address: event.user?.ip_address ?? '',
-        name: event.user?.name ?? '',
-        email: event.user?.email ?? '',
-      }}
-      // this is the subheading for the avatar, so displayEmail in this case is a misnomer
-      displayEmail={pathname}
-    />
-  );
-}
+const SubHeading = styled('div')`
+  font-size: ${p => p.theme.fontSizeMedium};
+  line-height: ${p => p.theme.text.lineHeightBody};
+  color: ${p => p.theme.gray300};
```

Comment:
We should use aliases rather than the color directly:

```suggestion
  color: ${p => p.theme.subText};
```